pkg/util/limitrange/limitrange.go

```diff
@@ -17,16 +17,18 @@ limitations under the License.
 package limitrange
 
 import (
-	"fmt"
-	"strings"
-
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	resourcehelpers "k8s.io/component-helpers/resource"
 
 	"sigs.k8s.io/kueue/pkg/util/resource"
 )
 
+const (
+	RequestsMustLessThanLimitRangeMaxMessage = "requests must less than limitRange max"
+	RequestsMustExceedLimitRangeMinMessage   = "requests must exceed limitRange min"
```

Comment:
```suggestion
        RequestsMustNotBeAboveLimitRangeMaxMessage = "requests must not be above the limitRange max"
	RequestsMustNotBeBelowLimitRangeMinMessage = "requests must not be below the limitRange min"
```
For example, "requests must less than limitRange max" might not be accourate, because equal max is also ok.

Reply:
Uhm, that sounds reasonable. Thank you for your suggestions!

Reply:
@mimowo Done, PTAL. Thanks!